### ISSUE TITLE
 0.00MH/s mining rate - can't mine

### ISSUE THREAD

**Issue by IAMtheIAM** [Genoil/cpp-ethereum#209]
I get 0.00mhz/s mine rate

Here's my flag - redacted personal info

`ethminer.exe --opencl-device --farm-recheck 200 -G -S west.mypool.org:0000 -O 0x(myworkerhash).myname/myemail`
```
setx GPU_FORCE_64BIT_PTR 0

SUCCESS: Specified value was saved.

setx GPU_MAX_HEAP_SIZE 100

SUCCESS: Specified value was saved.

setx GPU_USE_SYNC_OBJECTS 1

SUCCESS: Specified value was saved.

setx GPU_MAX_ALLOC_PERCENT 100

SUCCESS: Specified value was saved.

setx GPU_SINGLE_ALLOC_PERCENT 100

SUCCESS: Specified value was saved.

ethminer.exe --opencl-device --farm-recheck 200 -G -S west.mypool.org:0000 -O 0x(myworkerhash)

Genoil's ethminer 0.9.41-genoil-1.1.7
=====================================================================
Forked from github.com/ethereum/cpp-ethereum
CUDA kernel ported from Tim Hughes' OpenCL kernel
With contributions from nicehash, nerdralph, RoBiK and sp_

Please consider a donation to:
ETH: 0xeb9310b185455f863f526dab3d245809f6854b4d

Found suitable OpenCL device [Bonaire ] with 4294967236 bytes of GPU memory
  i  13:05:49|main  Connecting to stratum server west.mypool.org:0000
  i  13:05:49|stratum  Connected to stratum server west.mypool.org : 0000
  i  13:05:49|stratum  Starting farm
  i  13:05:50|stratum  Subscribed to stratum server
  i  13:05:50|stratum  Authorized worker 0x(myworkerhash)
  i  13:05:50|stratum  Received new job #af624f73
  i  13:05:50|openclminer0  set work; seed: #3828abc5, target: #00000000dbe5
  i  13:05:50|openclminer0  Initialising miner...
  m  13:05:50|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:51|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:51|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:51|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:51|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:51|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:52|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:52|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:52|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:52|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
Using platform: AMD Accelerated Parallel Processing
Using device: Bonaire(OpenCL 2.0 AMD-APP (1800.5))
  m  13:05:52|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:53|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:53|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:53|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:53|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:53|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:54|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:54|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:54|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:54|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:54|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  m  13:05:55|main  Mining on PoWhash #af534f73 : 0.00MH/s [A0+0:R0+0:F0]
  i  13:05:55|stratum  Received new job #87cece46

```
Benchmark also fails: hangs on trial 5

```
ethminer -G -M
Genoil's ethminer 0.9.41-genoil-1.1.7
=====================================================================
Forked from github.com/ethereum/cpp-ethereum
CUDA kernel ported from Tim Hughes' OpenCL kernel
With contributions from nicehash, nerdralph, RoBiK and sp_

Please consider a donation to:
ETH: 0xeb9310b185455f863f526dab3d245809f6854b4d

Found suitable OpenCL device [Bonaire ] with 4294967236 bytes of GPU memory
Benchmarking on platform: CL
Preparing DAG for block #0
Warming up...
  i  13:22:34|openclminer0  set work; seed: #00000000, target: #000000000000
  i  13:22:34|openclminer0  Initialising miner...
Using platform: AMD Accelerated Parallel Processing
Using device: Bonaire(OpenCL 2.0 AMD-APP (1800.5))
Trial 1... 0
Trial 2... 0
Trial 3... 0
Trial 4... 0
Trial 5... 0

```
Using win10x64. AMD FirePro W5100 card 4GB

**Comment by progone** [comment]
I have similar issue, on solo mine rig

<h6>sudo ethminer -M -X --opencl-platform 1
[sudo] password for matthew: 
ethminer: /usr/local/cuda-8.0/targets/x86_64-linux/lib/libOpenCL.so.1: no version information available (required by ethminer)
ethminer: /usr/local/cuda-8.0/targets/x86_64-linux/lib/libOpenCL.so.1: no version information available (required by /usr/local/lib/libethcore.so)
ethminer: /usr/local/cuda-8.0/targets/x86_64-linux/lib/libOpenCL.so.1: no version information available (required by /usr/local/lib/libethash-cl.so)
ethminer: /usr/local/cuda-8.0/targets/x86_64-linux/lib/libOpenCL.so.1: no version information available (required by /usr/local/lib/libethash-cl.so)
Genoil's ethminer 0.9.41-genoil-1.1.7
=====================================================================
Forked from github.com/ethereum/cpp-ethereum
CUDA kernel ported from Tim Hughes' OpenCL kernel
With contributions from nicehash, nerdralph, RoBiK and sp_ 

Please consider a donation to:
ETH: 0xeb9310b185455f863f526dab3d245809f6854b4d

modprobe: ERROR: ../libkmod/libkmod-module.c:832 kmod_module_insert_module() could not find module by name='fglrx_updates'
modprobe: ERROR: could not insert 'fglrx_updates': Unknown symbol in module, or unknown parameter (see dmesg)
Error! Fail to load fglrx kernel module! Maybe you can switch to root user to load kernel module directly
modprobe: ERROR: ../libkmod/libkmod-module.c:832 kmod_module_insert_module() could not find module by name='fglrx_updates'
modprobe: ERROR: could not insert 'fglrx_updates': Unknown symbol in module, or unknown parameter (see dmesg)
Error! Fail to load fglrx kernel module! Maybe you can switch to root user to load kernel module directly
modprobe: ERROR: ../libkmod/libkmod-module.c:832 kmod_module_insert_module() could not find module by name='fglrx_updates'
modprobe: ERROR: could not insert 'fglrx_updates': Unknown symbol in module, or unknown parameter (see dmesg)
Error! Fail to load fglrx kernel module! Maybe you can switch to root user to load kernel module directly
[OPENCL]:Found suitable OpenCL device [GeForce GTX 1070] with 8506769408 bytes of GPU memory
Benchmarking on platform: CUDA
Preparing DAG for block #0
Warming up...
Trial 1... 0
Trial 2... 0
Trial 3... 0
Trial 4... 0
Trial 5... 0
min/mean/max: 0/0/0 H/s
inner mean: 0 H/s
</h6>

**Comment by progone** [comment]
```
 nvidia-smi -q

==============NVSMI LOG==============

Timestamp                           : Wed Jun 14 22:08:00 2017
Driver Version                      : 375.66

Attached GPUs                       : 1
GPU 0000:03:00.0
    Product Name                    : GeForce GTX 1070
    Product Brand                   : GeForce
    Display Mode                    : Enabled
    Display Active                  : Enabled
    Persistence Mode                : Disabled
    Accounting Mode                 : Disabled
    Accounting Mode Buffer Size     : 1920
    Driver Model
        Current                     : N/A
        Pending                     : N/A
    Serial Number                   : N/A
    GPU UUID                        : GPU-6d1545b1-f361-ad89-3a2a-663e05ca7920
    Minor Number                    : 0
    VBIOS Version                   : 86.04.26.00.80
    MultiGPU Board                  : No
    Board ID                        : 0x300
    GPU Part Number                 : N/A
    Inforom Version
        Image Version               : G001.0000.01.03
        OEM Object                  : 1.1
        ECC Object                  : N/A
        Power Management Object     : N/A
    GPU Operation Mode
        Current                     : N/A
        Pending                     : N/A
    GPU Virtualization Mode
        Virtualization mode         : None
    PCI
        Bus                         : 0x03
        Device                      : 0x00
        Domain                      : 0x0000
        Device Id                   : 0x1B8110DE
        Bus Id                      : 0000:03:00.0
        Sub System Id               : 0x85A01043
        GPU Link Info
            PCIe Generation
                Max                 : 2
                Current             : 2
            Link Width
                Max                 : 16x
                Current             : 16x
        Bridge Chip
            Type                    : N/A
            Firmware                : N/A
        Replays since reset         : 0
        Tx Throughput               : 0 KB/s
        Rx Throughput               : 28000 KB/s
    Fan Speed                       : 28 %
    Performance State               : P0
    Clocks Throttle Reasons
        Idle                        : Not Active
        Applications Clocks Setting : Active
        SW Power Cap                : Not Active
        HW Slowdown                 : Not Active
        Sync Boost                  : Not Active
        Unknown                     : Not Active
    FB Memory Usage
        Total                       : 8112 MiB
        Used                        : 1227 MiB
        Free                        : 6885 MiB
    BAR1 Memory Usage
        Total                       : 256 MiB
        Used                        : 5 MiB
        Free                        : 251 MiB
    Compute Mode                    : Default
    Utilization
        Gpu                         : 0 %
        Memory                      : 1 %
        Encoder                     : 0 %
        Decoder                     : 0 %
    Encoder Stats
        Active Sessions             : 0
        Average FPS                 : 0
        Average Latency             : 0 ms
    Ecc Mode
        Current                     : N/A
        Pending                     : N/A
    ECC Errors
        Volatile
            Single Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                Total               : N/A
            Double Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                Total               : N/A
        Aggregate
            Single Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                Total               : N/A
            Double Bit            
                Device Memory       : N/A
                Register File       : N/A
                L1 Cache            : N/A
                L2 Cache            : N/A
                Texture Memory      : N/A
                Texture Shared      : N/A
                Total               : N/A
    Retired Pages
        Single Bit ECC              : N/A
        Double Bit ECC              : N/A
        Pending                     : N/A
    Temperature
        GPU Current Temp            : 47 C
        GPU Shutdown Temp           : 99 C
        GPU Slowdown Temp           : 96 C
    Power Readings
        Power Management            : Supported
        Power Draw                  : 39.79 W
        Power Limit                 : 151.00 W
        Default Power Limit         : 151.00 W
        Enforced Power Limit        : 151.00 W
        Min Power Limit             : 75.00 W
        Max Power Limit             : 170.00 W
    Clocks
        Graphics                    : 1582 MHz
        SM                          : 1582 MHz
        Memory                      : 4006 MHz
        Video                       : 1430 MHz
    Applications Clocks
        Graphics                    : N/A
        Memory                      : N/A
    Default Applications Clocks
        Graphics                    : N/A
        Memory                      : N/A
    Max Clocks
        Graphics                    : 1987 MHz
        SM                          : 1987 MHz
        Memory                      : 4004 MHz
        Video                       : 1708 MHz
    Clock Policy
        Auto Boost                  : N/A
        Auto Boost Default          : N/A
    Processes
        Process ID                  : 1045
            Type                    : G
            Name                    : /usr/lib/xorg/Xorg
            Used GPU Memory         : 729 MiB
        Process ID                  : 2246
            Type                    : G
            Name                    : compiz
            Used GPU Memory         : 492 MiB
        Process ID                  : 23813
            Type                    : G
            Name                    : /usr/lib/firefox/firefox
            Used GPU Memory         : 1 MiB

```

**Comment by progone** [comment]
@IAMtheIAM @Genoil  Good news, I am up and running

I use Ubuntu 16.04,  I found a solution in the Nvidia setting,  this video gave me enough hints to open the nvidia setting via terminal.  The video is based on Windows thou.... but Nvidia hierarchy is the same for both systems.  Good luck [Nvidia Settings](https://youtu.be/hCriJL07zic)

**Comment by IAMtheIAM** [comment]
Why closed? I never received any help to get this resolved.

**Comment by Genoil** [comment]
I closed everything from the 0.9.41-1.x.x series as I can't give support any longer. I have reforked from the chfast fork which is much improved. Haven't got binaries in place yet, will hopefully follow soon.

**Comment by getkey** [comment]
Does that mean that your improvements are gone or you just merged @chfast's stuff into your fork?
